### PR TITLE
PDA-82 Form Widgets

### PR DESCRIPTION
**Pull request by ffleandro**
Added Support for the following form widgets:
    Combo, Slider, TextArea, Radio, Double, Date, MultipleSelector and InputList;

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: src/main/java/org/entando/plugins/pda/pam/service/process/KieFormDeserializer.java [hunk 1]
```diff
@@ -75,8 +113,9 @@ public Form deserialize(JsonParser jsonParser, DeserializationContext deserializ
         return result.get(0);
     }
 
+    @SuppressWarnings("PMD.CyclomaticComplexity")
```
Comment (werbth) [inline src/main/java/org/entando/plugins/pda/pam/service/process/KieFormDeserializer.java:116]: We need to find a way to solve this cyclomatic complexity

File: src/main/java/org/entando/plugins/pda/pam/service/process/KieFormDeserializer.java [hunk 2]
```diff
@@ -162,24 +237,45 @@ private Integer getInteger(JsonNode node, String label) {
...
+        JsonNode value = getNode(node, label);
+        return value == null ? null : value.asDouble();
+    }
+
     private Boolean getBoolean(JsonNode node, String label) {
         JsonNode value = getNode(node, label);
         return value != null && value.asBoolean();
     }
 
+    @SuppressWarnings("PMD.CyclomaticComplexity")
```
Comment (werbth) [inline src/main/java/org/entando/plugins/pda/pam/service/process/KieFormDeserializer.java:250]: We should solve this cyclomatic complexity.

**Comment by werbth** [comment]
Retest this please

**Comment by entando-service** [comment]
All is well

**Review by werbth — APPROVED** [review]

**Pull request merged**